next.config.js

```diff
@@ -2,6 +2,10 @@
 const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
-}
+  env: {
```

Comment:
Hi! According to next documentation (https://nextjs.org/docs/basic-features/environment-variables) the recommended way to add env variables is in the `.env.local` file

Reply:
I have added .env.local file but you know its a private file so git do not push this file, Create .env.local file and paste `APP_LOGO=JuaniGallo
APP_NAME=NotionBlog` in it. and env variables must be configured in Next.js which I have already configured.
<img width="800" alt="Screenshot 2022-09-22 at 8 13 28 AM" src="https://user-images.githubusercontent.com/58821374/191650554-6cf7a8d0-1129-4ade-8d1a-e23eb0597910.png">

Reply:
Awesome, then let's remove the key env from this file. As it's not needed after Next 9.4

Reply:
**`.env.local`** provides values to backend
**`next.config.js`** provides values to frontend
If I delete it then the app wouldn't be able to show it in UI.

Reply:
https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser